packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -30,7 +33,7 @@ abstract class OverlayRoute<T> extends Route<T> {
     navigator.overlay?.insertAll(_overlayEntries, above: insertionPoint);
   }
 
-  // Subclasses shouldn't call this if they want to delay the finished() call.
+  /// Subclasses shouldn't call this if they want to delay the finished() call.
```

Comment:
does this overwrite the inherited documentation? if so, we should duplicate the inherited documentation here, and adjust it as necessary to explain the thing with finished().